File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// TopNews SDK utility: prepare_body

class TopNewsPrepareBody
{
    public static function call(TopNewsContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: go/utility/prepare_body.go
package utility

import "github.com/voxgig-sdk/top-news-sdk/core"

func prepareBodyUtil(ctx *core.Context) any {
	op := ctx.Op

	if op.Input == "data" {
		body := ctx.Utility.TransformRequest(ctx)
		return body
	}

	return nil
}
